CondCore/CondDB/test/condTestRegression.py

```diff
@@ -9,13 +9,25 @@
 import re
 import json
 
-readers = { 'CMSSW_9_0_1'        : ['slc6_amd64_gcc530'], 
+# Boost 1.51 [no writers available - a back-port is required!]
+#readers = { 'CMSSW_7_1_0'        : ['slc6_amd64_gcc490'], 
+#          }
+
+# Boost 1.57
+#readers = { 'CMSSW_7_5_0'        : ['slc6_amd64_gcc491'], 
+#          }
+
+# Boost 1.63
+#readers = { 'CMSSW_9_0_0'        : ['slc6_amd64_gcc530'], 
+#          }
+
+# Boost 1.67 [No reference release yet...]
+readers = {
           }
 
-writers = { 'CMSSW_9_0_1'        : [ ('slc6_amd64_gcc530', 'ref901-s6530.db') ],
-            'CMSSW_8_1_0'        : [ ('slc6_amd64_gcc530', 'ref810-s6530.db') ],
-            'CMSSW_8_0_26'       : [ ('slc6_amd64_gcc530', 'ref8026-s6530.db')],
-            'CMSSW_7_6_6'        : [ ('slc6_amd64_gcc493', 'ref766-s6493.db')],
+writers = { 'CMSSW_9_0_1'        : [ ('slc6_amd64_gcc630', 'ref901-s6630.db')],
+            'CMSSW_8_1_0'        : [ ('slc6_amd64_gcc530', 'ref750-s6530.db'),('slc6_amd64_gcc600', 'ref750-s600.db')],
```

Comment:
@ggovi for my understanding, why this change compared to the previous version where both 8_0_26 and 8_1_0 were appearing with gcc530?

Reply:
901 and 810 makes more sense since they are respectively the first release built with gcc630 and gcc530. I honestly don't remember why we had the 8_0_26, I suspect was simply taken the release used to develop the test.